client/src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import ApolloClient from 'apollo-boost';
 import { ApolloProvider } from 'react-apollo'; // binds apollo to React
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { StickyContainer, Sticky } from 'react-sticky';
```

Comment:
Are we using the react-sticky package? Can you please remove the react-sticky install ```npm uninstall react-sticky``` if we are not.